pkg/commands/resolver.go

```diff
@@ -127,11 +138,19 @@ func makeBuilder(ctx context.Context, bo *options.BuildOptions) (*build.Caching,
 	return build.NewCaching(innerBuilder)
 }
 
+// MakePublisher creates a ko publisher
+func MakePublisher(po *options.PublishOptions) (publish.Interface, error) {
+	return makePublisher(po)
```

Comment:
Same comment about `makePublisher` vs `MakePublisher`, `makeBuilder`, etc. -- if we're going to export them, I'd prefer to just export them.

Reply:
As above. I was thinking of an expand-and-contract style of approach (https://www.martinfowler.com/bliki/ParallelChange.html).